app/lib/features/space/settings/widgets/space_settings_menu.dart

```diff
@@ -31,16 +32,35 @@ class SpaceSettingsMenu extends ConsumerWidget {
           children: [
             ...spaceProfile.when(
               data: (spaceProfile) => [
-                SpaceParentBadge(
-                  roomId: spaceId,
-                  badgeSize: 18,
-                  child: ActerAvatar(
+                canonicalParent.maybeWhen(
```

Comment:
same as before. if all we do is show a different default in `null` why bother with a `maybeWhen` and not just use `canoncialParent.valueOrNull`?!?